### PR TITLE
Fixes to Meade Longitude and UC handling

### PR DESCRIPTION
**Pull request by ClutchplateDude**
- Corrected Longitude parsing to account for sign
- Corrected Longitude output to provide sign
- Corrected inverted UTC offset
- Corrected handshake response to 0x06 to be P for Polar mode
- EEPROM debug code caused crash due to using F() macro in args
- Added specific Longitude output function for Meade
- Fixed comment on Meade :SC command

### CODE REVIEW

File: src/EPROMStore.cpp [hunk 1]
```diff
@@ -121,9 +121,9 @@ void EEPROMStore::displayContents()
     uint16_t marker = readUint16(MAGIC_MARKER_AND_FLAGS_ADDR);
     LOG(DEBUG_EEPROM, "[EEPROM]: Magic Marker: %x", marker);
 
-    LOG(DEBUG_INFO, "[EEPROM]: Values? %s", ((marker & MAGIC_MARKER_MASK) == MAGIC_MARKER_VALUE) ? F("Yes") : F("No"));
-    LOG(DEBUG_INFO, "[EEPROM]: Extended values? %s", ((marker & EXTENDED_FLAG) == EXTENDED_FLAG) ? F("Yes") : F("No"));
-    LOG(DEBUG_INFO, "[EEPROM]: IsPresent(EXTENDED)? %s", (EEPROMStore::isPresent(EXTENDED_FLAG) ? F("Yes") : F("No")));
+    LOG(DEBUG_INFO, "[EEPROM]: Values? %s", ((marker & MAGIC_MARKER_MASK) == MAGIC_MARKER_VALUE) ? "Yes" : "No");
```
Comment (julianneswinoga) [inline src/EPROMStore.cpp:124]: Why're you removing `F()`?

File: src/DayTime.cpp [hunk 2]
```diff
@@ -294,6 +295,11 @@ const char *DayTime::formatStringImpl(char *targetBuffer, const char *format, ch
                     {
                         switch (macro)
```
Comment (julianneswinoga) [inline src/DayTime.cpp:296]: More of a general question, but why are we doing our own string formatting? Is this stuff specific to an interface?

**Review by julianneswinoga — CHANGES_REQUESTED** [review]

File: src/EPROMStore.cpp (hunk 1 above)
Comment (ClutchplateDude) [reply src/EPROMStore.cpp:124]: Caused a crash in the arguments list. I don't know why, but after removing it, it no longer crashed.

File: src/DayTime.cpp (hunk 2 above)
Comment (ClutchplateDude) [reply src/DayTime.cpp:296]: Not sure what you mean? As opposed to using sprintf()?

File: src/EPROMStore.cpp (hunk 1 above)
Comment (julianneswinoga) [reply src/EPROMStore.cpp:124]: Huh, good to know I guess :+1:

File: src/DayTime.cpp (hunk 2 above)
Comment (julianneswinoga) [reply src/DayTime.cpp:296]: Yeah, the standard library of string formatting functions

Comment (ClutchplateDude) [reply src/DayTime.cpp:296]: That was two years ago, I can't remember yesterdays breakfast.... there was a reason in that I couldn't get it to work correctly on the Arduino. I don't remember why, though.

Comment (julianneswinoga) [reply src/DayTime.cpp:296]: Lol fair enough, if it's not interface specific I may look into removing this stuff and changing over to standard string formatting

**Review by julianneswinoga — APPROVED** [review]

**Pull request merged**